labman/gui/templates/library_pooling.html

```diff
@@ -0,0 +1,358 @@
+{% extends sitebase.html %}
+
+{% block head %}
+<link rel="stylesheet" href="/static/vendor/css/slick-default-theme.css" type="text/css"/>
+<link rel="stylesheet" href="/static/vendor/css/slick.grid.css" type="text/css"/>
+
+<script src="/static/vendor/js/jquery.event.drag-2.3.0.js" type="text/javascript"></script>
+<script src="/static/vendor/js/slick.core.js" type="text/javascript"></script>
+<script src="/static/vendor/js/slick.grid.js" type="text/javascript"></script>
+<script src="/static/vendor/js/slick.editors.js" type="text/javascript"></script>
+
+<script type='text/javascript'>
+
+  var poolParams = {
+    'min': [{'prefix': 'floor-vol-', 'value': '100', 'desc': 'volume for low conc samples (nL):', 'min': '1', 'step': '1'},
```

Comment:
same comment as before regarding maintaining sync with these parameters with what's in the server side code

Reply:
I am moving this code to the python handler and adding a unittest that will check the consistency between these two. That is the best that I can do - would that work?